xml/System.Security.Cryptography.X509Certificates/X509AuthorityInformationAccessExtension.xml

```diff
@@ -1,308 +1,308 @@
+        <param name="caIssuersUris">A collection of CAIssuers URI values to embed in the extension.</param>
+        <param name="critical">
+          <see langword="true" /> if the extension is critical; otherwise, <see langword="false" />.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Security.Cryptography.X509Certificates.X509AuthorityInformationAccessExtension" /> class from a collection of OCSP and CAIssuer values.</summary>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentException">Both <paramref name="ocspUris" /> and <paramref name="caIssuersUris" /> are
+              either <see langword="null" /> or empty.</exception>
+        <exception cref="T:System.Security.Cryptography.CryptographicException">One of the values in <paramref name="ocspUris" /> or <paramref name="caIssuersUris" />
+              contains characters outside of the International Alphabet 5 (IA5) character space
+              (which is equivalent to 7-bit US-ASCII).</exception>
+      </Docs>
+    </Member>
+    <Member MemberName="CopyFrom">
+      <MemberSignature Language="C#" Value="public override void CopyFrom (System.Security.Cryptography.AsnEncodedData asnEncodedData);" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig virtual instance void CopyFrom(class System.Security.Cryptography.AsnEncodedData asnEncodedData) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.Security.Cryptography.X509Certificates.X509AuthorityInformationAccessExtension.CopyFrom(System.Security.Cryptography.AsnEncodedData)" />
+      <MemberSignature Language="VB.NET" Value="Public Overrides Sub CopyFrom (asnEncodedData As AsnEncodedData)" />
+      <MemberSignature Language="F#" Value="override this.CopyFrom : System.Security.Cryptography.AsnEncodedData -&gt; unit" Usage="x509AuthorityInformationAccessExtension.CopyFrom asnEncodedData" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; override void CopyFrom(System::Security::Cryptography::AsnEncodedData ^ asnEncodedData);" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Security.Cryptography</AssemblyName>
+        <AssemblyVersion>7.0.0.0</AssemblyVersion>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="asnEncodedData" Type="System.Security.Cryptography.AsnEncodedData" />
+      </Parameters>
+      <Docs>
+        <param name="asnEncodedData">The <xref data-throw-if-not-resolved="true" uid="System.Security.Cryptography.AsnEncodedData"></xref> to be copied.</param>
+        <summary>Copies the extension properties of the specified <xref data-throw-if-not-resolved="true" uid="System.Security.Cryptography.AsnEncodedData"></xref> object.</summary>

```

Comment:
This seems like the only effective change. Why is the file showing +308 -308 in the diff? Same for all other files.

Reply:
It looks like a significant number of the changes are whitespace only changes.

Reply:
Yes, sorry about that. I configured the tool to try to normalize modified files to a single kind of endline.
Can you please review the PR with "ignore whitespace" enabled?